src/components/search/Search.jsx

```diff
@@ -118,7 +122,17 @@ const Search = () => {
             onClose();
           }}
         />
-        {canEnrollWithEnterpriseOffers && hasLowEnterpriseOffersBalance && <EnterpriseOffersLowBalanceAlert />}
+
+        { canEnrollWithEnterpriseOffers && shouldDisplayBalanceAlert && (
+          <EnterpriseOffersBalanceAlert
+            adminText={balanceAlertAttributes.adminText}
+            alertClassName={balanceAlertAttributes.className}
+            alertVariant={balanceAlertAttributes.variant}
+            alertIcon={balanceAlertAttributes.icon}
+            alertHeading={balanceAlertAttributes.heading}
+            alertText={balanceAlertAttributes.text}
```

Comment:
nit: Follow-up from Slack DM, I do think spreading on `...generateBalanceAlertAttributes(hasNoEnterpriseOffersBalance)` would be nice here :)